app/handlers/analysis.py

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env python
+import os
+import subprocess
+import shutil
 
 import tornado.web
 
+from traffic_cloud_utils.app_config import get_project_path, get_project_video_path, update_config_without_sections, get_config_without_sections
+from traffic_cloud_utils.emailHelper import EmailHelper
+from objectTracking import objectTrack
+from safetyAnalysis import safetyAnalysis
```

Comment:
I don't think you can import these functions like this. These functions are methods of the ObjectTrackingHandler class, so they aren't in the top level available for import.